portal/app/core/assets/js/skycons.js

```diff
@@ -0,0 +1,726 @@
+(function(global) {
```

Comment:
Please remove this file and add it from the cdn. We don't want this code in our codebase nor do we want to maintain it.
Here's a link for example: https://cdnjs.cloudflare.com/ajax/libs/skycons/1396634940/skycons.min.js

Reply:
Done